.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
     - tip
     - 1.8
-    - 1.7
```

Comment:
Is dropping 1.7 necessary here?

Reply:
No but why are we still doing it while everyone else has dropped it. runc/moby/cri-o ...
1.8 is supported on RHEL.